SmartTutor/ContentModel/LearningObjects/Challenges/FulfillmentStrategy/ProjectChecker/ProjectAnalyzer.cs

```diff
@@ -0,0 +1,70 @@
+﻿using CodeModel.CaDETModel.CodeItems;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SmartTutor.ContentModel.LearningObjects.Challenges.FulfillmentStrategy.ProjectChecker
+{
+    public class ProjectAnalyzer : ChallengeFulfillmentStrategy
```

Comment:
ProjectChecker might be more suitable to adhere to the convention of the other strategies.